src/NuGet.Core/NuGet.Packaging/Licenses/NuGetLicenseData.cs

```diff
@@ -112,135 +117,142 @@ public static class NuGetLicenseData
-            { "GPL-2.0-only", new LicenseData(licenseID: "GPL-2.0-only", isOsiApproved: true, isDeprecatedLicenseId: false, isFsfLibre: true) },
-            { "GPL-2.0-or-later", new LicenseData(licenseID: "GPL-2.0-or-later", isOsiApproved: true, isDeprecatedLicenseId: false, isFsfLibre: true) },
+            { "GPL-2.0", new LicenseData(licenseID: "GPL-2.0", isOsiApproved: true, isDeprecatedLicenseId: true, isFsfLibre: false) },
+            { "GPL-2.0+", new LicenseData(licenseID: "GPL-2.0+", isOsiApproved: true, isDeprecatedLicenseId: true, isFsfLibre: false) },
+            { "GPL-2.0-only", new LicenseData(licenseID: "GPL-2.0-only", isOsiApproved: true, isDeprecatedLicenseId: false, isFsfLibre: false) },
+            { "GPL-2.0-or-later", new LicenseData(licenseID: "GPL-2.0-or-later", isOsiApproved: true, isDeprecatedLicenseId: false, isFsfLibre: false) },
             { "GPL-2.0-with-autoconf-exception", new LicenseData(licenseID: "GPL-2.0-with-autoconf-exception", isOsiApproved: false, isDeprecatedLicenseId: true, isFsfLibre: false) },
             { "GPL-2.0-with-bison-exception", new LicenseData(licenseID: "GPL-2.0-with-bison-exception", isOsiApproved: false, isDeprecatedLicenseId: true, isFsfLibre: false) },
             { "GPL-2.0-with-classpath-exception", new LicenseData(licenseID: "GPL-2.0-with-classpath-exception", isOsiApproved: false, isDeprecatedLicenseId: true, isFsfLibre: false) },
             { "GPL-2.0-with-font-exception", new LicenseData(licenseID: "GPL-2.0-with-font-exception", isOsiApproved: false, isDeprecatedLicenseId: true, isFsfLibre: false) },
             { "GPL-2.0-with-GCC-exception", new LicenseData(licenseID: "GPL-2.0-with-GCC-exception", isOsiApproved: false, isDeprecatedLicenseId: true, isFsfLibre: false) },
-            { "GPL-3.0", new LicenseData(licenseID: "GPL-3.0", isOsiApproved: true, isDeprecatedLicenseId: true, isFsfLibre: true) },
-            { "GPL-3.0+", new LicenseData(licenseID: "GPL-3.0+", isOsiApproved: true, isDeprecatedLicenseId: true, isFsfLibre: true) },
-            { "GPL-3.0-only", new LicenseData(licenseID: "GPL-3.0-only", isOsiApproved: true, isDeprecatedLicenseId: false, isFsfLibre: true) },
-            { "GPL-3.0-or-later", new LicenseData(licenseID: "GPL-3.0-or-later", isOsiApproved: true, isDeprecatedLicenseId: false, isFsfLibre: true) },
+            { "GPL-3.0", new LicenseData(licenseID: "GPL-3.0", isOsiApproved: true, isDeprecatedLicenseId: true, isFsfLibre: false) },
+            { "GPL-3.0+", new LicenseData(licenseID: "GPL-3.0+", isOsiApproved: true, isDeprecatedLicenseId: true, isFsfLibre: false) },
+            { "GPL-3.0-only", new LicenseData(licenseID: "GPL-3.0-only", isOsiApproved: true, isDeprecatedLicenseId: false, isFsfLibre: false) },
+            { "GPL-3.0-or-later", new LicenseData(licenseID: "GPL-3.0-or-later", isOsiApproved: true, isDeprecatedLicenseId: false, isFsfLibre: false) },
```

Comment:
I think something must have gone wrong while importing? I am very very very surprised that the GPLv3 would not be considered Libre by the FSF?

Reply:
note: This seems to be happening to a lot of other ones.

Reply:
The data source is broken: https://github.com/spdx/LicenseListPublisher/issues/102. 

There are a few license additions here. I'll clean-up the data manually for this change.

Reply:
Cleaned it up manually.

Note that some of the deprecated licenses don't have the isFsfLibre data any more.
I have left it as false, because when a license is deprecated, it's not relevant whether it's osi or libre.